### ISSUE TITLE
Support for Directory Output

### ISSUE THREAD

**Issue by serkolch** [webpack-contrib/file-loader#91]
For reasons outside my control, we have a single webpack output path to a directory (e.g. app/scripts) - I can't do multiple configs for different entries / modules. 

I'm using file-loader to hash images. Ideally, they'd end up in a folder outside of scripts (e.g. app/static), so I've used:

```
module: {  
    loaders: [  
        {  
            test: /\.(png|jpg|jpeg|gif|svg)$/,  
            loader: "file?name=../static/[md5:hash:hex:8][name].[ext]"
        }  
    ]  
}  
```

This creates the hashed images into the correct directory, but upon requiring the image files, the html shows the '../static' in the src:

`<img src="../static/131ddf32imagefile.png>`

Is there support for file-loader to choose the output directory? I've tried overwriting that with public path, but it just prepends the public path value to "../static".

**Comment by serkolch** [comment]
Conversely, is this something that could be solved by passing a function in as the publicPath? How would that be done?

**Comment by shri3k** [comment]
I'm running into the same issue too. It's great if your site actually hosts static files in `/static` and you actually have `/static` directory which works nicely but if it's other than `/static` like say `/public` or just `/` (default in `express`) then you'll be forced to host your static files in root or create a directory that plays well with just how `file-loader` creates directory.

**Comment by evilebottnawi** [comment]
@serkolch Is the problem still there? Did comment https://github.com/webpack-contrib/file-loader/issues/91#issuecomment-255459977 solved your problem? If not please create minimum reproducible test repo, thanks!

**Comment by michael-ciniawsky** [comment]
Fixed by #81
